app/controllers/nonsipp/common/package.scala

```diff
@@ -37,6 +37,9 @@ package object common {
       case IdentitySubject.LoanRecipient =>
         loansmadeoroutstanding.saveProgress(srn, index, userAnswers, nextPage, alwaysCompleted)
 
+      case IdentitySubject.SharesSeller =>
```

Comment:
It looks good @MadStu .  One general question though: do you need to populate the progress data (as completed) in `SharesPrePopulationProcessor` ?